includes/js/komoju-checkout-blocks.js

```diff
@@ -51,7 +49,7 @@ const KomojuPaymentModule = (() => {
                         };
                     }
 
-                    function submitFields(fields) {
+                    async function submitFields(fields) {
```

Comment:
Great catch!

I just thought we need to carefully test both successful payments and failing payments, because making this function really `async` could change the timing this function returns, and thus could result in another timing issues. It's not blocking this PR but I expect you would double-check the behaviour manually. 🙏

Reply:
Thank you for the check!

I've updated the submitFields function based on the feedback. I removed the unnecessary async keyword from the Promise constructor to clarify the function's control flow and error handling, which should help avoid any potential timing issues.